docs/releases/1.9.2.txt

```diff
@@ -71,3 +71,7 @@ Bugfixes
 
 * Fixed a crash when using a reverse ``OneToOneField`` in
   ``ModelAdmin.readonly_fields`` (:ticket:`26060`).
+
+* Fixed a crash when calling the ``migrate`` command in a test case with the
```

Comment:
yeah I realized the test was a bit convoluted but the planner bug seemed legitimate to me, I just can't find another way to trigger it.